home/import_content_pages.py

```diff
@@ -224,7 +226,8 @@ def clean_excel_cell(cell_value: str | float | datetime | None) -> str:
             for name, cell in zip(header, row):  # noqa: B905 (TODO: strict?)
                 if name and cell:
                     r[name] = clean_excel_cell(cell)
-            rows.append(ContentRow.from_flat(r))
+            if r:
+                rows.append(ContentRow.from_flat(r))
```

Comment:
Does this work? Maybe we need a test? I'm expecting `r` here to still be `{"title": "", ...}` for blank rows, unless blank rows get returned as `[]`?

Reply:
Added a test :-)